src/index.html

```diff
@@ -22,6 +22,347 @@
+          <span class="card__price-name">Price:</span>
+          <span class="card__price-price">$2,199</span>
+        </div>
+        <a
+          href="#"
+          class="card__buy"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div class="card">
+        <div class="card__image"></div>
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__stars-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-name">Price:</span>
+          <span class="card__price-price">$2,199</span>
+        </div>
+        <a
+          href="#"
+          class="card__buy"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div class="card">
+        <div class="card__image"></div>
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__stars-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-name">Price:</span>
+          <span class="card__price-price">$2,199</span>
+        </div>
+        <a
+          href="#"
+          class="card__buy"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div class="card">
+        <div class="card__image"></div>
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__stars-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <span class="card__price-name">Price:</span>
+          <span class="card__price-price">$2,199</span>
+        </div>
+        <a
+          href="#"
+          class="card__buy"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
The product card structure is repeated multiple times. Consider using a templating engine or JavaScript to dynamically generate these cards to improve maintainability and reduce redundancy.